src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1629,10 +1629,13 @@ class S6Importer final : public IParkImporter
 
     std::string GetUserString(rct_string_id stringId)
     {
-        const auto originalString = _s6.custom_strings[(stringId - USER_STRING_START) % 1024];
-        std::string_view originalStringView(originalString, USER_STRING_MAX_LENGTH);
-        auto withoutFormatCodes = RCT12::RemoveFormatCodes(originalStringView);
-        return rct2_to_utf8(withoutFormatCodes, RCT2_LANGUAGE_ID_ENGLISH_UK);
+        const auto originalStringC = _s6.custom_strings[(stringId - USER_STRING_START) % 1024];
+        std::string_view originalString(originalStringC, USER_STRING_MAX_LENGTH);
```

Comment:
Diff would look cleaner if you hadn't touched these lines

Reply:
Done, thanks!